tests/common/osutil/test_default.py

```diff
@@ -657,16 +658,19 @@ def set_command(command, output='', exit_code=0):
         wait = FlexibleVersion(version) >= osutil._IPTABLES_LOCKING_VERSION
         uid = 42
 
-        version_command = set_command(osutil._get_iptables_version_command(), output=str(version))
-        list_command = set_command(osutil._get_firewall_list_command(wait), output="Mock Output")
-        set_command(osutil._get_firewall_packets_command(wait))
-        set_command(osutil._get_firewall_drop_command(wait, "-C", destination))
-        set_command(osutil._get_firewall_drop_command(wait, "-A", destination))
-        set_command(osutil._get_firewall_accept_command(wait, "-A", destination, uid))
+        version_command = set_command(osutil.get_iptables_version_command(), output=str(version))
+        list_command = set_command(osutil.get_firewall_list_command(wait), output="Mock Output")
+        set_command(osutil.get_firewall_packets_command(wait))
+        set_command(osutil.get_firewall_drop_command(wait, AddFirewallRules.CHECK_COMMAND, destination))
+        set_command(osutil.get_firewall_drop_command(wait, AddFirewallRules.APPEND_COMMAND, destination))
+        set_command(osutil.get_firewall_accept_command(wait, AddFirewallRules.APPEND_COMMAND, destination, uid))
+        set_command(osutil.get_accept_tcp_rule("-w" if wait else "", AddFirewallRules.APPEND_COMMAND, destination))
```

Comment:
This check `"-w" if wait else ""` will be relevant for all the functions passing `wait`, maybe add this to all?

Reply:
Addressed

Reply:
Instead of having a copy of the same statement, maybe make it a variable and pass that to each function? 

```python
wait_cmd = "-w" if wait else ""
set_command(osutil.get_firewall_packets_command(wait_cmd))
        set_command(osutil.get_firewall_drop_command(wait_cmd, AddFirewallRules.CHECK_COMMAND, destination))
```